src/test/kotlin/com/zepben/evolve/services/network/NetworkServiceUtilsTest.kt

```diff
@@ -134,8 +134,12 @@ internal class NetworkServiceUtilsTest {
         isSeriesCompensator: (SeriesCompensator) -> String,
         isGround: (Ground) -> String,
         isGroundDisconnector: (GroundDisconnector) -> String,
+        isGroundingImpedance: (GroundingImpedance) -> String,
```

Comment:
todo: Should be updated to match the revised order of the class under test.

Reply:
As in folder and alphabetically based on test files?

Reply:
`NetworkServiceUtilsTest` is a test for `NetworkServiceUtils`. You made changes to the order of these parameters in `NetworkServiceUtils` that have not been reflected in `NetworkServiceUtilsTest`

Reply:
Changed and added missing verifyInvoke() from SeriesCompensator onwards.